### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by numerodiciannove**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17]),
+])
```
Comment (vsmutok) [inline app/test_main.py:14]: add tests for negative numbers and invalid data types

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Review by Grayder0152 — APPROVED** [review]
